### DISCUSSION TITLE
Resolve conflicts with a pull request (I am a noob at this)

### DISCUSSION THREAD

**Discussion by will2asher** [community/community#23384] [New to GitHub]
<p>I am at this page is my repository:</p><aside class="onebox githubpullrequest" data-onebox-src="https://github.com/will2asher/RubberBand/pull/1">
  <header class="source">

      <a href="https://github.com/will2asher/RubberBand/pull/1" target="_blank" rel="noopener nofollow ugc">github.com/will2asher/RubberBand</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Pull Request">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 12 16" aria-hidden="true"><path d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/will2asher/RubberBand/pull/1" target="_blank" rel="noopener nofollow ugc">Rubberband pre-release update </a>
    </h4>

    <div class="branches">
      <code>will2asher:master</code> ← <code>angband:master</code>
    </div>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-06-29" data-time="23:30:15" data-timezone="UTC">11:30PM - 29 Jun 21 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/will2asher" target="_blank" rel="noopener nofollow ugc">
          <img alt="will2asher" src="https://user-images.githubusercontent.com/81661183/181076686-bbe91451-71cd-4435-86a8-616b502b3f45.jpeg" class="onebox-avatar-inline" width="20" height="20">
          will2asher
        </a>
      </div>

      <div class="lines" title="209 commits changed 302 files with 20809 additions and 7933 deletions">
        <a href="https://github.com/will2asher/RubberBand/pull/1/files" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+20809</span>
          <span class="removed">-7933</span>
        </a>
      </div>
    </div>
  </div>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
I downloaded github desktop to cherry pick and resolve conflicts, but I click on the link where it says “You can also [open this in GitHub Desktop]”. An error message comes up in the app that says “A branch with that name already exists”. How do I resolve this?</p>

**Comment by DutytoDevelop** [comment, marked as answer]
<p>Hello <a class="mention" href="/will2asher">@will2asher</a>,</p>
<p>According to others that have experienced the same issue, it looks like the branch ‘master’ is clashing since there’s already a ‘master’ branch, however I could be wrong. <a class="mention" href="/shiftkey">@shiftkey</a> explains the issue <a href="https://github.com/desktop/desktop/issues/4527#issuecomment-385590392" rel="noopener nofollow ugc">here</a>. I’ll put his quote below:</p>
<blockquote>
<p>Yeah, there’s repetitions of the same error in here:</p>
<pre><code class="lang-auto">2018-04-25T12:43:38.725Z - error: [ui] `git -c credential.helper= checkout --progress overleaf/master -b master --` exited with an unexpected code: 128.
fatal: A branch named 'master' already exists.
</code></pre>
<p>I gather that the existing <code>master</code> is tracking <code>origin/master</code> or something similar.</p>
<p>A potential solution for this would be to, in cases where an existing local branch exists and will clash, use the convention <code>{remote}-{branch}</code> when the user wants to checkout the same branch from another remote.</p>
</blockquote>
<p>I would think that changing the branch would fix the error that you’re experiencing. Would you try changing the branch name to something other than ‘master’ and see if that fixes the error? Here’s a <a href="https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/renaming-a-branch" rel="noopener nofollow ugc">guide</a> on GitHub on how to do that.</p>
<p>Let me know if this fixes the error that you’re having or if you’re still experiencing the same issue.</p>
<p>Thanks,<br>
Nicholas Hickam</p>

**Comment by will2asher** [comment]
<p>That fixes that issue.  (I hope I don’t need to make a new thread for me next questions.)<br>
I’m trying to cherry pick commits in Github Desktop, I click ‘open in editor’ to open the file in Visual Studio (when I thought I would be able to do this in Github Desktop), but it isn’t showing me what changes I need to resolve. How I do I see the changes I need to resolve?</p>
